animeal/src/Flows/Main/Modules/MoreModule/MorePartition/MorePartitionViewModel.swift

```diff
@@ -40,6 +40,15 @@ final class MorePartitionViewModel: MorePartitionViewModelLifeCycle,
                     self.coordinator.routeTo(.error(error.localizedDescription))
                 }
             }
+        case .deleteAccount:
+            self.model.handleDeleteUser { result in
```

Comment:
[weak self]

Reply:
nit: What do you think it would be good to show the pop up like "Are you sure you want to delete your account?" before calling that api?

Reply:
Done

Reply:
> nit: What do you think it would be good to show the pop up like "Are you sure you want to delete your account?" before calling that api?
It's works exactly as you suggest, delete account action called after Ok button is called